src/phoenix/trace/v1/utils.py

```diff
@@ -520,7 +520,10 @@ def _maybe_timestamp(obj: Optional[datetime]) -> Optional[Timestamp]:
 
 def _as_struct(obj: Mapping[str, Any]) -> Struct:
     struct = Struct()
-    struct.update(obj)
+    for key, value in obj.items():
+        if isinstance(value, Iterable) and not isinstance(value, (str, list, Mapping)):
+            value = list(value)
+        struct[key] = value
```

Comment:
Maybe safer to explicitly check for `ndarray` in this case?

Reply:
mainly it's because i don't want it as a dependency. i think `Iterable` is pretty safe given the exclusions

Reply:
i changed it up a bit